src/commands/serverCommand.ts

```diff
@@ -790,6 +792,37 @@ export async function executeQuery(
   isPython: boolean,
   isWorkbook: boolean,
   isFromConnTree?: boolean,
+): Promise<void> {
+  await window.withProgress(
+    {
+      cancellable: true,
+      location: ProgressLocation.Window,
+      title: `Executing Query (${executorName})`,
```

Comment:
Will we standardise the capitalisation, if "scratchpad" is lowercase then should "query" be lowercase too?